client/containers/Login/logout.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { LogOutUser } from '../../store/actions/auth/login';
+
+export class LogOut extends Component {
+  componentWillMount = () => {
+    // eslint-disable-next-line no-shadow
+    const { history, LogOutUser } = this.props;
+    LogOutUser();
```

Comment:
Kindly use Camel case or simply `logoutz
```suggestion
    logoutUser();
```